salt/memcached/memcached.conf

```diff
@@ -0,0 +1,49 @@
+########################################
+# File managed by salt state memcached #
+########################################
+
+# Run memcached as a daemon. This command is implied, and is not needed for the
+# daemon to run. See the README.Debian that comes with this package for more
+# information.
+-d
+
+# Log memcached's output to /var/log/memcached
+logfile /var/log/memcached.log
+
+# Be verbose
+# -v
+
+# Be even more verbose (print client commands as well)
+-vv
```

Comment:
This is probably not intended to be on by default?

Reply:
Removed. Good catch